webapp/models.py

```diff
@@ -85,6 +85,20 @@ class Webpage(db.Model, DateTimeMixin):
     reviewers = relationship("Reviewer", back_populates="webpages")
     jira_tasks = relationship("JiraTask", back_populates="webpages")
 
+    def to_dict(self):
```

Comment:
@britneywwc what does this method do?

Reply:
It converts the ORM object to a dictionary, otherwise the object cannot be jsonified and passed through the endpoint response. It isn't expected in the `/set-owner` output but I included it for checking purposes. I realise now that maybe the setter endpoint shouldn't return a response body with data and that should be a getter function instead. Let me know if I should remove this and just return 200 on a successful POST.

Reply:
in this particular line you are adding it to a "Webpage" class. Are you returning webpage object from `/set-owner` endpoint?

Reply:
You can exclude returning an object from the setter endpoint. The user object will exist on the FE since that's what's being sent to the endpoint as an input.

Reply:
Okay, I updated it to remove any objects from the setter endpoint